lib/python_helper_sympy.py

```diff
@@ -204,3 +246,37 @@ def point_to_error(s, ind, w = 5):
     w_left = ind - max(0, ind-w)
     w_right = min(ind+w, len(s)) - ind
     return s[ind-w_left:ind+w_right] + '\n' + ' '*w_left + '^' + ' '*w_right
+
+def sympy_to_json(a, allow_complex=True):
+    const = _Constants()
+
+    # Get list of variables in the sympy expression
+    variables = [str(v) for v in a.free_symbols]
+
+    # Check that variables do not conflict with reserved names
+    reserved = {**const.helpers, **const.variables, **const.hidden_variables, **const.functions}
+    if allow_complex:
+        reserved = {**reserved, **const.complex_variables, **const.hidden_complex_variables}
+    for k in reserved.keys():
+        for v in variables:
+            if k == v:
+                raise ValueError('sympy expression has a variable with a reserved name: {:s}'.format(k))
```

Comment:
Not that I mind the use of `format()`, but I've been starting to convert everything to f-string syntax: https://cito.github.io/blog/f-strings/

Reply:
good to know. next time.